Tests/Common/SimulatedBasicsStation.cs

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWan.Tests.Common
+{
+
+    using System;
+    using System.Text.Json;
+    using LoRaWan.Tests.Simulation.Models;
+    using Websocket.Client;
+    using System.IO;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using System.Runtime.InteropServices;
+    using LoRaWan.NetworkServer;
+    using LoRaTools.LoRaMessage;
+    using LoRaTools.Utils;
+    using System.Net.WebSockets;
+    using Xunit;
+    using System.Linq;
+
+    public sealed class SimulatedBasicsStation : IDisposable
+    {
+        private readonly string stationEUI;
+        private WebsocketClient DataWebsocketClient { get; set; }
```

Comment:
Why use a library for this and not something that [.NET gives us](https://docs.microsoft.com/en-us/dotnet/api/system.net.websockets.clientwebsocket?view=net-6.0)?

Reply:
Created #1221 to track the issue, this is a good idea but would require too much changes in this PR